lib/internal/test_runner/runner.js

```diff
@@ -724,8 +724,10 @@ function run(options = kEmptyObject) {
     runFiles = () => {
       root.harness.bootstrapPromise = null;
       root.harness.buildPromise = null;
+      let workerId = 1;
       return SafePromiseAllSettledReturnVoid(testFiles, (path) => {
-        const subtest = runTestFile(path, filesWatcher, opts);
+        const subtest = runTestFile(path, filesWatcher, opts, workerId);
```

Comment:
if I have 8 cpus and 16 tests, I would expect each worker to be reused twice, but this implementation will not reuse worker ids, I think that is misleading